File: JavaScript/tabel-perkalian.js
const prompt = require("prompt-sync")({ sigint: true });

let panjang = parseInt(prompt("Masukkan panjang tabel perkalian: "));
let lebar = parseInt(prompt("Masukkan lebar tabel perkalian: "));

process.stdout.write("*\t");
for (let i = 1; i <= panjang; i++) {
	process.stdout.write(i + "\t");
}
console.log();
for (let i = 1; i <= lebar; i++) {
	process.stdout.write(i + "\t");
	for (let j = 1; j <= panjang; j++) {
		process.stdout.write(i * j + "\t");
	}
	console.log();
}

File: C++/tabel-perkalian.cpp
#include <iostream>

using namespace std;

int main()
{
    int panjang, lebar;

    cout << "Masukkan panjang tabel perkalian: ";
    cin >> panjang;
    cout << "Masukkan lebar tabel perkalian: ";
    cin >> lebar;

    cout << "*\t";
    for (int i = 1; i <= panjang; i++)
    {
        cout << i << "\t";
    }
    cout << endl;
    for (int i = 1; i <= lebar; i++)
    {
        cout << i << "\t";
        for (int j = 1; j <= panjang; j++)
        {
            cout << (i * j) << "\t";
        }
        cout << endl;
    }

    return 0;
}